### PR TITLE
Fix/#143 fix planted trees in basket

### LINKED ISSUE #143: Fix planted trees in basket not handling properly
## Goal

Have the planted trees show proper default value and perists correctly

### PR DESCRIPTION
**Pull request by Kwandes**
## Goal

Have the planted trees show proper default value and persists correctly

## Requirements

___
✅ Closes: #143

### CODE REVIEW

File: apps/webstore/src/app/pages/auth/basket/basket.component.html [hunk 1]
```diff
@@ -76,30 +76,30 @@ <h2 class="desktop-title opacity">|</h2>
                         </div>
                     </div>
 
-                    <div class="donate-tree-option">
+                    <div class="plante-tree-option">
                         <button
```
Comment (Teodor25) [inline apps/webstore/src/app/pages/auth/basket/basket.component.html:80]: you wrote plante-tree-option without a d in planted.

File: apps/webstore/src/app/pages/auth/basket/basket.component.scss [hunk 2]
```diff
@@ -131,7 +131,7 @@
   text-align: right;
 }
 
-.donate-tree-option {
+.plante-tree-option {
```
Comment (Teodor25) [inline apps/webstore/src/app/pages/auth/basket/basket.component.scss]: with d

File: apps/webstore/src/app/pages/auth/basket/basket.component.spec.ts [hunk 3]
```diff
@@ -30,42 +30,42 @@ describe('BasketComponent', () => {
...
     expect(component).toBeTruthy();
   });
 
-  it('should increase donated trees quantity', () => {
-    expect(component.donatedTrees).toEqual(1);
+  it('should increase planted trees quantity', () => {
+    expect(component.plantedTrees).toEqual(1);
     const increaseButton = fixture.debugElement.query(
-      By.css('#donate-extra-trees-btn')
+      By.css('#plante-extra-trees-btn')
```
Comment (Teodor25) [inline apps/webstore/src/app/pages/auth/basket/basket.component.spec.ts]: with d

**Review by Teodor25 — CHANGES_REQUESTED** [review]
quick change needed: planted is written without a d at the end (plante) in some files.

File: apps/webstore/src/app/pages/auth/basket/basket.component.html (hunk 1 above)
Comment (Teodor25) [reply apps/webstore/src/app/pages/auth/basket/basket.component.html:80]: there are multiple places in the file where theres missing a d, so just do a command + f and replace it.

File: apps/webstore/src/app/pages/auth/basket/basket.component.spec.ts (hunk 3 above)
Comment (Kwandes) [reply apps/webstore/src/app/pages/auth/basket/basket.component.spec.ts]: Resolved in 9e025fa

File: apps/webstore/src/app/pages/auth/basket/basket.component.scss (hunk 2 above)
Comment (Kwandes) [reply apps/webstore/src/app/pages/auth/basket/basket.component.scss]: Resolved in 9e025fa

File: apps/webstore/src/app/pages/auth/basket/basket.component.html (hunk 1 above)
Comment (Kwandes) [reply apps/webstore/src/app/pages/auth/basket/basket.component.html:80]: Resolved in 9e025fa

File: apps/webstore/src/i18n/messages.dk.xlf [hunk 4]
```diff
@@ -1108,7 +1108,7 @@
       </trans-unit>
       <trans-unit id="payment-success-text" datatype="html">
         <source>Shortly you should recieve an order confirmation on your e-mail. We thank you for buying at Treecreate.dk, and for planting a tree. Your support is more important now, than it has ever been. If you wish to plant more trees, please visit:</source>
-        <target>Om et øje blik skulle du gerne modtage en ordrerbekræftelse på din e-mail. Vi takker dig for at have købt hos Treecreate.dk, og for at plante et træ. Din støtte har aldrig været vigtigere. Hvis du ønsker at plante flere træer, har du mulighed for at gøre det her: </target> 
+        <target>Om et øje blik skulle du gerne modtage en ordrerbekræftelse på din e-mail. Vi takker dig for at have købt hos Treecreate.dk, og for at planted et træ. Din støtte har aldrig været vigtigere. Hvis du ønsker at planted flere træer, har du mulighed for at gøre det her: </target>
```
Comment (Teodor25) [inline apps/webstore/src/i18n/messages.dk.xlf]: please dont 😂

**Review by Teodor25 — CHANGES_REQUESTED** [review]

**Review by Teodor25 — APPROVED** [review]

**Pull request merged**